benchmarks/Cargo.toml

```
@@ -10,17 +10,14 @@ publish = false
 roaring = { path = ".." }
 
 [dev-dependencies]
+once_cell = "1.9.0"
+git2 = "0.13.25"
```

This is frustrating that I can't compile due to the fact that OpenSSL isn't reachable, I prefer that we use this feature. This is also the reason why I prefer using `Command` with `git` and `unzip` as the benchmarks are used by developers and having `git` and `un/zip` is quite common for developpers.

```suggestion
git2 = { version = "0.13.25", features = ["vendored-openssl"] }
```

---

Looking at git2 `Cargo.toml`

```toml
[features]
unstable = []
default = ["ssh", "https", "ssh_key_from_memory"]
ssh = ["libgit2-sys/ssh"]
https = ["libgit2-sys/https", "openssl-sys", "openssl-probe"]
vendored-libgit2 = ["libgit2-sys/vendored"]
vendored-openssl = ["openssl-sys/vendored", "libgit2-sys/vendored-openssl"]
ssh_key_from_memory = ["libgit2-sys/ssh_key_from_memory"]
zlib-ng-compat = ["libgit2-sys/zlib-ng-compat"]
```

I've disabled all features now, so If I'm not mistaken: no default `https`, so no transitive `openssl-sys`

---

I just tried with `git2 = { version = "0.13.25", default-features = false }` and indeed it works great!